ocs_ci/utility/utils.py

```diff
@@ -974,6 +974,52 @@ def get_csi_versions():
     return csi_versions
 
 
+def parse_pgsql_logs(data):
+    """
+    Parse the pgsql benchmark data from ripsaw and return
+    the data in list format
+
+    Args:
+        data (str): log data from pgsql bench run
+
+    Returns:
+        list_data (list): data digestable by scripts with below
+                        format
+            eg: ( with only one item in the list)
+            [{'num_clients': '2', 'num_threads': '7', 'latency_avg': '7',
+             'lat_stddev': '0', 'tps_incl': '234', 'tps_excl': '243'}]
+    """
+
+    match = re.findall(
+        r'\[\{\'number_.*?\'number_of_transactions_per_client\':\s+\w+}\]',
+        data
+    )
+    pgsql_data = dict()
```

Comment:
Should this be initialized inside of the for loop?

Reply:
yeah will fix that, thx